src/main/resources/application.yaml

```diff
@@ -64,13 +64,15 @@ spring:
     database-platform: org.hibernate.dialect.PostgreSQLDialect
     properties:
       hibernate:
+        order_by.default_null_ordering: ${DEFAULT_NULL_ORDERING:last}
```

Comment:
Shouldn't this be separate from this hibernate settings? It could be confusing. We should add the add point details in the front to recognise this property.

Reply:
NA.